tests/test_target.py

```diff
@@ -1,4 +1,5 @@
 import unittest
+from unittest.mock import MagicMock
```

Comment:
Holy Mocks, batman! This is great, thank you for getting the mocks started in our program. This will up our testing game.

Reply:
Thanks!  It was great having Matt's mock test as reference too.